packages/headless/src/controllers/facets/facet/headless-facet.ts

```diff
@@ -96,15 +96,15 @@ export function buildFacet(
     ...controller,
     facetSearch: createFacetSearch(),
     /**
-     * Selects (deselects) the passed value if unselected (selected).
-     * @param selection The facet value to select or deselect.
+     * Toggles the passed value.
+     * @param selection (FacetValue) The facet value to select or deselect.
      */
     toggleSelect: (selection: FacetValue) =>
       dispatch(executeToggleFacetSelect({facetId: options.facetId, selection})),
     /**
      * Returns `true` if the passed facet value is selected and `false` otherwise.
-     * @param {FacetValue} The facet value to check.
-     * @returns {boolean}.
+     * @param value (FacetValue) The facet value to check.
+     * @returns Whether the passed facet value is selected.
```

Comment:
Are you sure the `@returns` here and below should not just be `boolean`? I think developers typically expect @returns to hold the return type.

Reply:
I was actually just going off all the examples of `@returns` already in the repo- for example, the `state` getter down below. I'll check with the docs team to see what we want here.